crates/tools/yml/src/main.rs

```diff
@@ -28,19 +27,35 @@ jobs:
       matrix:
         include:
           - version: stable
+            host: x86_64-pc-windows-msvc
             target: x86_64-pc-windows-msvc
+            etc:
           - version: nightly
+            host: x86_64-pc-windows-msvc
             target: i686-pc-windows-msvc
+            etc:
           - version: nightly
+            host: x86_64-pc-windows-gnu
             target: x86_64-pc-windows-gnu
+            etc:
           - version: stable
+            host: x86_64-pc-windows-gnu
             target: i686-pc-windows-gnu
```

Comment:
Was it intentional to now cross-compile x86_64 -> i686?

Reply:
It's not required but works for me.